### ISSUE TITLE
cross-fade between texture zooms to maintain texture density

### ISSUE THREAD

**Issue by meetar** [tangrams/tangram#612]
### Problem

When using a texture in a material, the default texture `scale` is `[1, 1]`, which means a texture isn't visible at native pixel density until z21 (on my retina screen):

http://tangrams.github.io/texture-demos/problem-case/
☝️ Zoom in until you see the textures scale properly, then adjust the slider and check out other zoom levels.

### Proposal

Here's a method of cross-fading between two copies of the texture for each zoom: 

http://tangrams.github.io/texture-demos/?url=scenes/texture-tiles.yaml#12/37.8090/-122.2220

This could be a more reasonable default scaling option for textures, maybe with a syntax addition of `scale: crossfade`.

**Comment by bcamper** [comment]
Seems like the proposal demo isn't currently loading.

**Comment by meetar** [comment]
Yep – fixed with https://github.com/tangrams/texture-demos/commit/a541532a2ed5a9193d38409ef21a7927b50728ee
